src/app/drive/services/network.service/index.ts

```diff
@@ -223,4 +223,33 @@ export function getEnvironmentConfig(isTeam?: boolean): EnvironmentConfig {
   };
 }
 
+export class Band {
+  private _startTime: number;
```

Comment:
We are already using `private` in TypeScript, the `_` convention is not required